traffic_signal_data/data/53189942.json

```diff
@@ -1,349 +0,0 @@
-{
```

Comment:
The changed geometry around this intersection caused a crosswalk to no longer get generated, so this previous override became invalid. I fixed it up manually, but then noticed the heuristics for two-stage actually produced something a little more reasonable than this override.

I've yet to regen all data; possibly this change to geometry will cause some unexpected gridlock somewhere to work through.